README.md

```diff
@@ -182,6 +182,7 @@ Table of Contents
   * [Conversion Tools](https://conversiontools.io/) - Online File Converter for documents, images, video, audio, eBooks. REST API is available. Libraries for Node.js, PHP, Python. Support files up to 50 GB (for paid plans). Free tier is limited by file size and number of conversions per day.
   * [CurlHub](https://curlhub.io) — Proxy service for inspecting and debugging API calls. Free plan includes 10,000 requests per month.
   * [CurrencyScoop](https://currencyscoop.com) - Realtime currency data API for fintech apps. Free plan includes 5000 calls per month.
+  * [Dagpi](https://dagpi.xyz) - A free api that features powerful image manipulation and more. Free plan includes 45 api requests per minute.
```

Comment:
trying to verify the 45/minute claim or even a commitment that its a long term free tier but cant find reference to that on the site, do you have a link?

Reply:
Hey so, you can join our discord server and ask. Although not official, it is our free limit. Paid customers get based on their tier, 60 , 90 or 120 requests per second. If you make a request, you get the x-Preliterate-Remainig header with requests left for that minute.

Reply:
There is over 1000 links on this list, do you think it's realistic that I join a discord server for each to verify the eligability?

I dont think so. Information has be available on the site in question to be listed.

Reply:
Okay sorry. I'll add pricing